caddy/README.md

```diff
@@ -8,6 +8,13 @@ tagline: |
 To update or switch versions, run `webi caddy@stable` (or `@v2.4`, `@beta`,
 etc).
 
+### Files
+
+```text
+~/.local/bin/caddy
+~/.local/opt/caddy-{version}
```

Comment:
I happen to know exactly where the config files for `caddy` are, so here's what this section should look like, verbatim:

```text
~/.config/envman/PATH.env
~/.local/bin/caddy
~/.config/caddy/autosave.json
~/.local/share/caddy/certificates/
<PROJECT-DIR>/Caddyfile
```